### DISCUSSION TITLE
Modifying the max route cost

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7766] [Q&A] [answered]
Hello,

What are potential serious consequences for modifying some constants in `mle_types.hpp`? For example, I wanted to test MLE Long Routes, but the `kMaxRouteCost` of 127 seems a little high, and I am worried it might use too much ram. I was thinking of decreasing it to 87 or 88 for a maximum of 22 hops, assuming each hop has a link cost of 4. Would that cause issues?

**Comment by jwhui** [comment, marked as answer]
Note that the `OPENTHREAD_CONFIG_MLE_LONG_ROUTES_ENABLE` feature is experimental and not conformant to the Thread Specification. That said, setting the max route cost to a lower value should not harm anything as long as you do not expect to have paths that exceed that limit.

As for the other constants in `mle_types.hpp`, they are typically specified in the Thread Specification. Changing those values would make the implementation non-conformant to the Thread Specification. The side-effect of changing a given value really depends on the constant being changed.
